docs/docs/spark-procedures.md

```diff
@@ -270,7 +270,8 @@ the `expire_snapshots` procedure will never remove files which are still require
 | `retain_last` |    | int       | Number of ancestor snapshots to preserve regardless of `older_than` (defaults to 1) |
 | `max_concurrent_deletes` |    | int       | Size of the thread pool used for delete file actions (by default, no thread pool is used) |
 | `stream_results` |    | boolean       | When true, deletion files will be sent to Spark driver by RDD partition (by default, all the files will be sent to Spark driver). This option is recommended to set to `true` to prevent Spark driver OOM from large file size |
-| `snapshot_ids` |   | array of long       | Array of snapshot IDs to expire. |
+| `snapshot_ids` |   | array of long       | Array of snapshot IDs to expire (note that the table's expiration properties will still be applied to remove all expired snapshots, unless `older_than` or `retain_last` arguments are also given). |
```

Comment:
Not sure this note is clearer. I'm thinking about words like `Additional array of snapshots IDs to expire, besides those expired by `older_than` and `retain_last` arguments, and table properties`.

Reply:
Thanks for the quick feedback!  Reworded.